.github/CONTRIBUTING.md

```diff
@@ -7,12 +7,11 @@ operating environment, schemachange version and python version. Whenever possibl
 also include a brief, self-contained code example that demonstrates the problem.
 
 We have
-included [issue templates for reporting bugs, requesting features and seeking clarifications.](https://github.com/Snowflake-Labs/schemachange/issues/new/choose)
-Choose the appropriate issue template to contribute to the repository.
+included [issue templates](https://github.com/Snowflake-Labs/schemachange/issues/new/choose) for reporting bugs, requesting features and seeking clarifications. Choose the appropriate issue template to contribute to the repository.
 
 ## Contributing code
 
-Thanks for your interest in contributing code to schemachange!
+Thank you for your interest in contributing code to schemachange!
 
 + If this is your first time contributing to a project on GitHub, please read through
   our [guide to contributing to schemachange](guide-to-contributing-to-schemachange).
```

Comment:
Is this meant to be a self-referencing link? It returns a 404 right now.

Reply:
It was missing a `#` to refer to the next section. Thanks for catching this.